scripts/build/compile

```diff
@@ -12,6 +12,9 @@ run_tests = False
 in_ci = os.getenv("CI") == "true"
 optimize = in_ci
 
+fsharp_thing_to_build = "fsdark.sln"
+# sometimes it's handy to only build a specific project
+# fsharp_thing_to_build = "src/LocalExec"
```

Comment:
This actually was really handy. Often when I'm doing a big refactor, I'll get things compiling _just enough_ for one project, then another, etc. And this is a way to keep .NET from trying to build 'too much'